### DISCUSSION TITLE
How to reload `.ideavimrc` on Windows?

### DISCUSSION THREAD

**Discussion by jtpavlock** [JetBrains/ideavim#353] [Q&A] [answered]
Hello,

I am wondering how I can reload the configuration file on Windows. It doesn't seem like `:source` works, and [there is no reload button](https://youtrack.jetbrains.com/issue/VIM-2254). Are there any alternatives?

**Comment by jtpavlock** [comment, marked as answer]
I figured out it works if you use the full path, and use forward slashes instead of the normal backslashes on Windows e.g.

```
source: C:/Users/jacob/.ideavimrc
```

**Reply by KL-Sven** [reply]
`so: ~/.ideavimrc` works on Windows, too.

**Comment by chylex** [comment]
Reload button is fixed by https://github.com/JetBrains/ideavim/pull/390.
